Marlin/src/module/configuration_store.cpp

```diff
@@ -1482,6 +1486,13 @@ void MarlinSettings::postprocess() {
         for (uint8_t q = MAX_EXTRUDERS * 2; q--;) EEPROM_READ(dummy);
       #endif
 
+      #if ENABLED(SOFT_ENDSTOPS_MENU_ITEM)
+		    EEPROM_READ(soft_endstops_enabled);
```

Comment:
Tabs are verboten

Reply:
Ohhhh god, i dont know why, but it looks like my editor problem. Sorry about that.(I am using osx Atom editor)